nicknamer/nicknamer.py

```diff
@@ -87,7 +90,12 @@ async def on_member_update(self, before, after):
             for e in settings:
                 if after.id in e:
                     if after.nick != e[1]:
-                        await after.edit(nick=e[1], reason="Nickname frozen.")
+                        try:
+                            await after.edit(nick=e[1], reason="Nickname frozen.")
+                        except discord.errors.Forbidden:
+                            log.info(
+                                f"Missing permissions to change {before.nick} ({before.id}) in {before.guild.id}"
+                            )
```

Comment:
Not sure if the nickname should be removed from config here and the "freezing" be aborted. What do you think?

Reply:
That's an even better idea. I was hesitant to impose any configuration adjusting but a config clear to reduce log spam when a server for some reason removes bot perms on after freeze set up would be solid. 

Aside, I will never understand why people actually do this (remove bot perms after setting up bot)

Reply:
I wouldn't wipe all of config. Just the one user it failed to change. User might have gone up in the hierarchy. That would also raise discord.Forbidden :)

Reply:
Oop touché!